lab-01-Jacob/gulpfile.js

```diff
@@ -0,0 +1,39 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+//make a command called lintMe
+gulp.task('lintMe', function() {
+  return gulp.src(['**/*.js', '!node_modules'])
+//the stuff gulp should look at is in the current directory or subdirectories with a .js extension, except from within the node_modules directory.
+  .pipe(eslint()) //run the eslint method on that stream of data
+  .pipe(eslint.format());//output the results to the console.
+});
+
+gulp.task('testMe', function() {
+  return gulp.src(['./test/*.js']) //look for any file in the test directory with a .js extension
+  .pipe(mocha());//run that stream of data through the mocha method
+});
+
+function nightsWatch() {
+  gulp.watch(['**/*.js', '!node_modules/**'], ['lintMe', 'testMe']);
+}
+// }//make a function called nightsWatch that makes gulp watch any file with a .js extension in the current directory or subdirectories (except node modules) and then run the lintMe and testMe tasks
+
+
+gulp.task('dev', function() {
+  nightsWatch();
```

Comment:
Rather than wrapping the `gulp.watch()` in a separate function scope, you can just put that line here inside this task.